From 525f66294291456a84d2c0a63c772b302868314f Mon Sep 17 00:00:00 2001
From: Thomas Wood <thomas.wood@intel.com>
Date: Fri, 27 Aug 2010 15:26:08 +0100
Subject: [PATCH 12/13] Don't build capplets not used by MeeGo

---
 capplets/Makefile.am |    6 +-----
 1 files changed, 1 insertions(+), 5 deletions(-)

diff --git a/capplets/Makefile.am b/capplets/Makefile.am
index b75b89a..4ba944f 100644
--- a/capplets/Makefile.am
+++ b/capplets/Makefile.am
@@ -1,16 +1,12 @@
 SUBDIRS = \
 	common			\
-	accessibility	\
 	appearance		\
-	default-applications	\
 	display			\
-	keybindings		\
 	keyboard		\
 	mouse			\
 	network			\
 	power			\
-	windows			\
-	date
+	date			\
 	language
 
 DIST_SUBDIRS = \
-- 
1.7.2.3

